### PR TITLE
[doc] Version no longer gets spellchecked

### PR DESCRIPTION
**Pull request by JJRcop**

### CODE REVIEW

File: doc/conf.py [hunk 1]
```diff
@@ -4,8 +4,13 @@
...
 # https://www.sphinx-doc.org/en/master/usage/configuration.html
 
-# -- Path setup --------------------------------------------------------------
-from pathlib import Path
+# -- Release import
+
+import sys, os
+sys.path.append(os.path.dirname(__file__))
+
+
```
Comment (quantum5) [inline doc/conf.py:12]:
```suggestion
```

File: doc/conf.py [hunk 2]
```diff
@@ -22,30 +27,17 @@
...
 .. |license| replace:: GPLv2
 """
 
+
 # -- General configuration ---------------------------------------------------
 
 # Add any Sphinx extension module names here, as strings. They can be
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
+
```
Comment (quantum5) [inline doc/conf.py:40]: Is there a reason for these new lines?

File: doc/lgrelease.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
+# -- Path setup --------------------------------------------------------------
+from pathlib import Path
+
+release = None
+
```
Comment (quantum5) [inline doc/lgrelease.py:5]: This variable is overwritten on all non-error paths.
```suggestion
```

File: doc/lgspell.py [hunk 4]
```diff
@@ -1,5 +1,9 @@
 #!/usr/bin/env python3
 import re
+import sys, os
+sys.path.append(os.path.dirname(__file__))
```
Comment (quantum5) [inline doc/lgspell.py:4]: Not necessary here.

**Review by gnif — CHANGES_REQUESTED** [review]
Please fix the white space issues, etc. This PR is rather important as ci builds are failing until it's done.

File: doc/conf.py (hunk 2 above)
Comment (JJRcop) [reply doc/conf.py:40]: mistake

Comment (JJRcop) [reply doc/conf.py:40]: makes my linter happy but can remove

**Pull request merged**